hs_core/tasks.py

```diff
@@ -374,6 +382,44 @@ def send_over_quota_emails():
             logger.debug('user ' + u.username + ' does not have UserQuota foreign key relation')
 
 
+@celery_app.task(ignore_result=True, base=HydroshareTask)
+def check_geoserver_registrations(resources):
+    # Check to ensure resources have updated web services registrations
+    if not resources:
+        cuttoff_time = timezone.now() - timedelta(days=1)
+        resources = BaseResource.objects.filter(updated__gte=cuttoff_time, raccess__public=True)
```

Comment:
Not all public resources use geoserver.  Are resources that do not contain geoserver files creating noise in the logging for failed_resources below?

Reply:
the web service middleware will return a successful response for resources that don't contain geospatial files